active/0003-new-config-files.md

```diff
@@ -95,21 +123,142 @@ to
 force_shutdown_policy: 10000|64MB
 ```
 
-The prior is not only more readable to users without reading the comments/documents, but also more intuitive to programmers for searching the config entry through the code files.
+The prior is not only more readable to users without reading the comments/documents, but also more
+intuitive to programmers for searching the config entry through the code files, as the ConfMap
+maintained in the memory is of the same structure (and name) as the config file.
+
+### The Structure of the Config Files
+
+At the beginning of the `emqx.conf` is the notes:
+
+```
+## NOTE: The configurations in this file will be overridden by `/etc/emqx_overrides.conf`
```

Comment:
I think it would be better to give the cover order explicitly.

Even without include emqx_ overrides.conf Each program has its own hard coded configuration item.

Like MySQL:

```
> mysql --help
...
Default options are read from the following files in the given order:
/etc/my.cnf /etc/mysql/my.cnf ~/.my.cnf
```

```
cat /etc/mysql/my.cnf
...
!includedir /etc/mysql/conf.d/
!includedir /etc/mysql/mysql.conf.d/
```

Reply:
As we've discussed, the priority of config files should be conventional not configurable. So we need no `include` instructions in `emqx.conf`.